GitHub Actions start to fail for CQLInputFormatIT on master

See https://github.com/JanusGraph/janusgraph/runs/2374067404#step:6:123

> Error:  testGeoshapeGetValues  Time elapsed: 7.058 s  <<< ERROR!
java.lang.IllegalStateException: java.net.BindException: Cannot assign requested address: Service 'sparkDriver' failed after 16 retries (on a random free port)! Consider explicitly setting the appropriate binding address for the service 'sparkDriver' (for example spark.driver.bindAddress for SparkDriver) to the correct binding address.
Caused by: java.util.concurrent.ExecutionException: java.net.BindException: Cannot assign requested address: Service 'sparkDriver' failed after 16 retries (on a random free port)! Consider explicitly setting the appropriate binding address for the service 'sparkDriver' (for example spark.driver.bindAddress for SparkDriver) to the correct binding address.
Caused by: java.net.BindException: Cannot assign requested address: Service 'sparkDriver' failed after 16 retries (on a random free port)! Consider explicitly setting the appropriate binding address for the service 'sparkDriver' (for example spark.driver.bindAddress for SparkDriver) to the correct binding address.

Related discussions:
https://github.com/mlflow/mlflow/issues/4229
https://github.com/apache/spark/pull/24559#issuecomment-814507882

--- a/janusgraph-cql/src/test/resources/cql-read.properties
+++ b/janusgraph-cql/src/test/resources/cql-read.properties
@@ -31,6 +31,7 @@ cassandra.input.widerows=true
 # SparkGraphComputer Configuration #
 ####################################
 spark.master=local[4]
+spark.driver.host=127.0.0.1
 spark.executor.memory=1g
 spark.serializer=org.apache.spark.serializer.KryoSerializer
 spark.kryo.registrator=org.janusgraph.hadoop.serialize.JanusGraphKryoRegistrator

